Mathlib/Analysis/SpecialFunctions/Pow/NNReal.lean

```diff
@@ -224,6 +281,47 @@ theorem lt_rpow_inv_iff {x y : ℝ≥0} {z : ℝ} (hz : 0 < z) : x < y ^ z⁻¹
 theorem rpow_inv_lt_iff {x y : ℝ≥0} {z : ℝ} (hz : 0 < z) : x ^ z⁻¹ < y ↔ x < y ^ z := by
   simp only [← not_le, le_rpow_inv_iff hz]
 
+section
+variable {y : ℝ≥0}
+
+lemma rpow_lt_rpow_of_neg (hx : 0 < x) (hxy : x < y) (hz : z < 0) : y ^ z < x ^ z :=
```

Comment:
It looks like all 12 lemmas here aren't specialisations to natural or integer powers. Please add this to the PR description, or put these in a separate PR.

Reply:
PR description updated